### ISSUE TITLE
Text review: Can whole topics be sent for re-review? 

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#335]
@ssin1901 @ilmartyrk @anettlinno Reading this in a moderation email I had a doubt whether this is correct (from what I understand of how the process works), so wanted to check that we haven't missed something from the flow or the instructions. 

Would any of you be able to share with me what the steps are for someone to send a topic for review by a moderator, after they have edited the topic following it being reported? I understand with an argument it takes them to a pop-up window that they can edit the argument and then send it for a review, but I wonder how it works with a whole topic that has lots of text in it, plus photos and attachments, and potentially lots of separate edits?  

(I tried to try it out myself in the test environment to see what happens, but came into [this issue](https://github.com/citizenos/citizenos-fe/issues/322) of not being able to make a topic public (and it is only public topics that can be reported)).  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/8883ccae-ea70-48ec-93eb-5283488eb633)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish yes, after topic has been hidden, editors can edit the content and submit the topic for review. Then a moderator can check if content is now acceptable and can "unhide" the reported topic

**Comment by BeccaMelhuish** [comment]
@ilmartyrk I tried this out as part of the email text review (getting the emails to fire by going through the motions), and it doesn't look like this is the case? 

When I went into the topic I was email to say was reported, following the link, nothing shows up for me - no indication that it's been reported and no way to report it, unless I'm missing it? 
![image](https://github.com/citizenos/citizenos-fe/assets/61320526/15a3c2e8-048f-4645-b9f7-c5fd3f7a0912)

**Comment by BeccaMelhuish** [comment]
P.S I see in your comment that you say "after topic has been hidden", but that's not what the email says should happen, and how the moderation system works as far as I understand: you're meant to be able to edit out the problem before a moderator has even  looked at it no? 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/9f475a91-98ac-4c45-ad57-f7010548159c)

**Comment by BeccaMelhuish** [comment]
How often are topics actually reported though? As if it's very rare I'll stop spending so much time on them for now, but we should have a review of the whole system when we have time I think (so phase II) - it's a bit rough around the edges :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish they are rally rare, I think I've seen only one, that was somebodies test from outside of our team

**Comment by BeccaMelhuish** [comment]
OK good to know! Would be good to get the system straight at some point, but not a priority for now then! Will add Phase II tag (though maybe a phase beyond if so rare).
